docs/migrating.rst

```diff
@@ -155,3 +155,39 @@ Upgrading:
 - Create a new *empty* directory, e.g. ``generated`` or ``lib/generated/proto`` etc.
 - Regenerate your python files into this directory
 - Update import statements, e.g. ``import ExampleMessage from generated``
+
+
+[2.0.0b4] to [2.x.x]
```

Comment:
I wasn't quite sure how to adhere to the format here.

Reply:
Hmm... It's a bit of a strange one since it's from one beta release to another.

Personally I would include it in the changelog but I'd like to know what others think.

Reply:
Yeah I'm inclined to agree here

Reply:
Are you saying move this entire migrating comment in to the changelog, or just delete it? Atm, I put a short line item in the changelog as well.

Reply:
I'd suggest just moving most of this to the change log

Reply:
https://github.com/danielgtaylor/python-betterproto/pull/311/commits/08069babf40d42651fb6d8bf005acb9c2e9923ec moves the migration notes to the changelog